src/components/usersTable/usersTable.js

```diff
@@ -0,0 +1,37 @@
+import {Table, TableBody, TableCell, TableContainer, TableHead, TableRow} from "@material-ui/core";
+
+
+export default function UsersTable({
+  users
+}) {
+  return (
+    <TableContainer>
+      <Table sx={{minWidth: 700}} aria-label="users table">
```

Comment:
try to avoid inline styles

Reply:
Didn't even know about [sx](https://v4.mui.com/system/basics/#sx-property) before 😅